### DISCUSSION TITLE
Fedora Agent Install

### DISCUSSION THREAD

**Discussion by HozerTech** [immense/Remotely#186] [General]
For others looking here are the prerequisites for installing the agent on Fedora (tested on F33), I assume it would also work on other RHEL based distros like CentOS but I have not tested it. 

for MS repos - 
`sudo rpm -Uvh https://packages.microsoft.com/config/rhel/8/packages-microsoft-prod.rpm`

then
`dnf install -y dotnet-runtime-3.1 libX11-devel glibc-devel libgdiplus libXtst-devel xclip `

I then removed lines 24 through 47 in the Install-Ubuntu-x64.sh script and ran the rest as is and it all works great.
the script can easily be modified, given the time in the next few days I will post it.

**Comment by locaIhost** [comment]
Have you heard of **debug** without candy?
Not for сanon...... 
catch ᛉ
